packages/contracts-ethers/README.md

```diff
@@ -24,12 +24,10 @@ const daoFactoryInstance = DAOFactory__factory.connect(...);
 ```ts
```

Comment:
I noticed that the contracts-ethers README is in bad shape to. We should create a task to update this.

Reply:
will be addressed in: OS-783